src/gsp.py

```diff
@@ -60,15 +73,15 @@ async def get_truths_for_a_specific_gsp(
     If regime is not specific, the latest gsp yield is loaded.
     """
 
-    logger.info(f"Get truth values for gsp id {gsp_id} and regime {regime}")
+    logger.info(f"Get truth values for gsp id {gsp_id} and regime {regime} for {user}")
```

Comment:
What is `user` here?
We don't want to be logging PII (Personally Identifiable Information), so maybe, if this `user` includes a full name and/or email, we could be logging the user `id` instead?